specification/maps/data-plane/DataRegistry/stable/2022-12-01/examples/DataRegistries_Patch.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "geography": "us",
+    "api-version": "2022-12-01",
+    "udid": "25084fb7-307a-4720-8f91-7952a0b91012",
+    "dataRegistryPatch": {
+      "connection": {
+        "blobPath": "/my/blob/path-new.json"
+      }
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Operation-Location": "https://us.atlas.microsoft.com/mapData/operations/{operationId}?api-version=2022-12-01",
```

Comment:
Maybe we could provide real example for operationId.

Reply:
Fixed